src/App.jsx

```diff
@@ -21,11 +21,10 @@ function App () {
       <div>How's everyone doing??</div>
       <div>{info}</div>
       <span>
-        <button onClick={() => setCount(count - 1)}>-</button>
-        {count}
+        <button onClick={() => setCount(count - 1)}>-</button> {count}{' '}
```

Comment:
```suggestion
        <button onClick={() => setCount(count - 1)}>-</button> {count}
```

You should probably get rid of this space